net-misc/frr/frr-7.5-r1.ebuild

```diff
@@ -66,46 +56,50 @@ src_prepare() {
 }
 
 src_configure() {
-	econf \
-		--disable-static \
-		--with-pkg-extra-version="-gentoo" \
-		--enable-configfile-mask=0640 \
-		--enable-logfile-mask=0640 \
-		--prefix=/usr \
-		--libdir=/usr/lib/frr \
-		--sbindir=/usr/lib/frr \
-		--libexecdir=/usr/lib/frr \
-		--sysconfdir=/etc/frr \
-		--localstatedir=/run/frr \
-		--with-moduledir=/usr/lib/frr/modules \
-		--enable-exampledir=/usr/share/doc/${PF}/samples \
-		--enable-user=frr \
-		--enable-group=frr \
-		--enable-vty-group=frr \
-		--enable-multipath=64 \
-		$(use_enable doc) \
-		$(use_enable fpm) \
-		$(use_enable grpc) \
-		$(use_enable ipv6 ospf6d) \
-		$(use_enable ipv6 ripngd) \
-		$(use_enable ipv6 rtadv) \
-		$(use_enable kernel_linux realms) \
-		$(use_enable nhrp nhrpd) \
-		$(usex ospfapi '--enable-ospfclient' '' '' '') \
-		$(use_enable rpki) \
-		$(use_enable snmp) \
+	local myconf=(
+		--disable-static
+		--with-pkg-extra-version="-gentoo"
+		--enable-configfile-mask=0640
+		--enable-logfile-mask=0640
+		--prefix=/usr
+		--libdir=/usr/lib/frr
+		--sbindir=/usr/lib/frr
+		--libexecdir=/usr/lib/frr
+		--sysconfdir=/etc/frr
+		--localstatedir=/run/frr
+		--with-moduledir=/usr/lib/frr/modules
+		--enable-exampledir=/usr/share/doc/${PF}/samples
+		--enable-user=frr
+		--enable-group=frr
+		--enable-vty-group=frr
+		--enable-multipath=64
+		$(use_enable doc)
+		$(use_enable fpm)
+		$(use_enable grpc)
+		$(use_enable ipv6 ospf6d)
+		$(use_enable ipv6 ripngd)
+		$(use_enable ipv6 rtadv)
+		$(use_enable kernel_linux realms)
+		$(use_enable nhrp nhrpd)
+		$(usex ospfapi '--enable-ospfclient' '' '' '')
+		$(use_enable rpki)
+		$(use_enable snmp)
 		$(use_enable systemd)
+	)
+	tc-is-cross-compiler && myconf+=( --with-clippy="/usr/bin/clippy" )
```

Comment:
`tc-is-cross-compiler && myconf+=( --with-clippy="${BROOT}"/usr/bin/clippy )`

Reply:
also, can't this be unconditional? if you're nor cross-compiling, BROOT will be empty

Reply:
missing `BROOT`?

Reply:
Yes, done now :)